main.nf

```diff
@@ -918,6 +1021,80 @@ if (!params.multipleSequencingRuns){
 			-o table/rel-feature-table.tsv --to-tsv
 		"""
 	}
+} else if (params.pacbio){
+	process dada_pacBio {
+		
+		publishDir "${params.outdir}", mode: 'copy',
+			saveAs: {filename -> 
+					 if (filename.indexOf("dada_stats.tsv") == 0)         "abundance_table/unfiltered/dada_stats.tsv"
+				else if (filename.indexOf("dada_report.txt") == 0)              "abundance_table/unfiltered/dada_report.txt"
+				else if (filename.indexOf("rel-feature-table.tsv") == 0)         "abundance_table/unfiltered/rel-feature-table.tsv"
+				else if (filename.indexOf("feature-table.tsv") == 0)             "abundance_table/unfiltered/feature-table.tsv"
+				else if (filename.indexOf("feature-table.biom") == 0)     "abundance_table/unfiltered/feature-table.biom"
+				else if (filename.indexOf("sequences.fasta") == 0)                      "representative_sequences/unfiltered/sequences.fasta"
+				else if (filename.indexOf("rep-seqs.qza") == 0)                 "representative_sequences/unfiltered/rep-seqs.qza"
+				else null}
+
+		input:
+		file demux from ch_dada_import
+		val trunc from ch_dada_trunc
+
+		output:
+		file("table.qza") into ch_qiime_table_raw
+		file("rep-seqs.qza") into (ch_qiime_repseq_raw_for_classifier,ch_qiime_repseq_raw_for_filter)
+		file("feature-table.tsv") into ch_tsv_table_raw
+		file("dada_stats.tsv")
+		file("feature-table.biom")
+		file("rel-feature-table.tsv")
+		file("sequences.fasta")
+		file("dada_report.txt")
+
+		when:
+		!params.untilQ2import
+
+		script:
+		"""
+		# Quality filtering with DADA2 filterAndTrim
+		# Might want to add params.minLen and params.maxLen in the future
```

Comment:
Totally agreed, this should be in parameters, could you add this? Just keep the values you added here as default.

Reply:
This seems resolved with the latest commit.